guides/getting-started/creating-your-own-server-instance.md

```diff
@@ -75,6 +75,8 @@ If you don't want to start from one of the recommended samples, you'll need to:
   ```csharp
   app.UseDeveloperExceptionPage();
 
+  app.UseForwardedHeaders();
```

Comment:
@johnxy84 I update your PR to ensure `app.UseForwardedHeaders()` appears before pretty much everything else, as it was added a bit too late in your original commit.